src/server/protocol.ts

```diff
@@ -2737,6 +2737,13 @@ namespace ts.server.protocol {
         payload: TypingsInstalledTelemetryEventPayload;
     }
 
+/* __GDPR__
+   "typingsinstalled" : {

```

Comment:
Looks like there might also be a `version` property.

Reply:
Where? I thought the properties only came from the interface itself.

Reply:
https://github.com/Microsoft/vscode/blob/34e7a220c87088afca5c57234c441b68576e8f77/extensions/typescript-language-features/src/utils/telemetry.ts#L42

Looks like the correct fix would be to reference `TypeScriptCommonProperties`.

Reply:
(For the other comment as well.)

Reply:
Unfortunately, this doesn't work right now. Filed Microsoft/vscode-gdpr-tooling#6 to track. I think I'll merge this anyway since we have existing metadata to fall back on.